src/triggers.js

```diff
@@ -125,24 +128,66 @@ export function addFunction(
   applicationId
 ) {
   add(Category.Functions, functionName, handler, applicationId);
-  add(Category.Validators, functionName, validationHandler, applicationId);
+  if (validationHandler) {
```

Comment:
You can remove the `if(validationHandler)`. If they are null the maybeRunValidation will just ignore it